spec/models/landing_page/block/statistics_spec.rb

```diff
@@ -129,4 +129,14 @@
       expect(subject.rows).to eq([])
     end
   end
+
+  context "with an unparseable attachment URL" do
```

Comment:
Minor: I think this should be inside the `describe #rows` block.

Reply:
Sorted